functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/project/NavSidebarSpec.groovy

```diff
@@ -0,0 +1,134 @@
+package org.rundeck.tests.functional.selenium.project
+
+import org.openqa.selenium.By
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.common.navigation.NavLinkTypes
+import org.rundeck.util.gui.pages.home.HomePage
+import org.rundeck.util.gui.pages.login.LoginPage
+import org.rundeck.util.gui.pages.project.SideBarPage
+import spock.lang.PendingFeatureIf
+import spock.lang.Unroll
+
+@SeleniumCoreTest
+class NavSidebarSpec extends SeleniumBase {
+    static final String SPEC_PROJECT = "NavSidebarSpec"
+
+    static final String ADMIN_USER = "NavSidebarSpec1"
+    static final String READONLY_USER = "NavSidebarSpec2"
+    static final String DELETE_USER = "NavSidebarSpec3"
+    static final String EXPORT_USER = "NavSidebarSpec4"
+    static final String IMPORT_USER = "NavSidebarSpec5"
+    static final String CONFIGURE_USER = "NavSidebarSpec6"
+    static final String ACL_READ_USER = "NavSidebarSpec7"
+    static final String ACL_ADMIN_USER = "NavSidebarSpec8"
+    static final String APP_ADMIN_USER = "NavSidebarSpec9"
+    static final String USER_PASS = "password"
+
+    def setupSpec() {
+        setupProject(SPEC_PROJECT)
+        //import acls
+        importSystemAcls("/NavSidebarSpec.aclpolicy", "NavSidebarSpec.aclpolicy")
+    }
+
+    def cleanupSpec() {
+        deleteProject(SPEC_PROJECT)
+        deleteSystemAcl("NavSidebarSpec.aclpolicy")
+    }
+
+    @Override
+    String toString() {
+        return super.toString()
+    }
+
+    /**
+     * All project settings navlinks
+     */
+    static List<NavLinkTypes> PROJ_SETTINGS_NAVLINKS = [
+        NavLinkTypes.EDIT_NODES,
+        NavLinkTypes.MOTD,
+        NavLinkTypes.README,
+        NavLinkTypes.PROJECT_CONFIG,
+        NavLinkTypes.ACCESS_CONTROL,
+        NavLinkTypes.SETUP_SCM,
+        NavLinkTypes.EXPORT_ARCHIVE,
+        NavLinkTypes.IMPORT_ARCHIVE,
+        NavLinkTypes.DELETE_PROJECT,
+    ]
+    /**
+     * Nav links visible with "configure" authorization
+     */
+    static List<NavLinkTypes> PROJ_SETTINGS_CONFIGURE_NAVLINKS = [
+        NavLinkTypes.PROJECT_CONFIG,
+        NavLinkTypes.STORAGE,
+        NavLinkTypes.EDIT_NODES,
+        NavLinkTypes.README,
+        NavLinkTypes.MOTD,
+        NavLinkTypes.SETUP_SCM
+    ]
+
+    @PendingFeatureIf(value = { data.username in [NavSidebarSpec.ADMIN_USER, NavSidebarSpec.APP_ADMIN_USER] }, reason = "fixed in later change")
```

Comment:
these cases are skipped in this PR, pending the changes brought in https://github.com/rundeck/rundeck/pull/9425